lib/src/chip/types.ts

```diff
@@ -42,6 +71,8 @@ type Props = {
    * Value of the tabindex attribute.
    */
   tabIndex?: number;
-};
+} & (PrefixIconProps | SuffixIconProps);

```

Comment:
I think this is not part of the `CommonProps` but part of the `Props` as you stated in the next line.

Reply:
Oh yes, it was a mistake to push this! Done!